zarf.schema.json

```diff
@@ -1060,18 +1084,19 @@
         },
         "vendor": {
           "type": "string",
-          "description": "Name of the distributing entity, organization or individual."
```

Comment:
Missing punctuation

Reply:
I just went ahead and added punctuation to all the json schema descriptions